layer/contracts/assets.py

```diff
@@ -11,7 +11,7 @@
 from layer.exceptions.exceptions import LayerClientException
 
 
-_ASSET_PATH_PATTERN = re.compile(
+_asset_path_PATTERN = re.compile(
```

Comment:
I think you regex should respect case = )

Reply:
Fixed!